src/main/java/it/gov/pagopa/payhub/activities/utility/Utilities.java

```diff
@@ -1,14 +1,24 @@
 package it.gov.pagopa.payhub.activities.utility;
 
+import lombok.Getter;
+
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
 public class Utilities {
-
     private Utilities(){}
     public static final Pattern EMAIL_PATTERN = Pattern.compile("^[a-zA-Z0-9_!#$%&'*+/=?`{|}~^.-]+@[a-zA-Z0-9.-]+$");
     public static final int IBAN_LENGTH = 27;
 
+    @Getter
+    public enum CLASSIFICATION {
```

Comment:
let's move it on a separate class
```suggestion
    public enum ClassificationsEnum {
```

Reply:
done